Jenkinsfile

```diff
@@ -41,20 +41,33 @@ timestamps {
           '''
```

Comment:
I have a feeling there is no point in running `Setup` stage with `yarn install` and `yarn setup` on PR since docker packages does not need it.